<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mi_detail_widget</class>
 <widget class="QWidget" name="mi_detail_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1029</width>
    <height>945</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>微逆编号</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="graph_tab">
      <attribute name="title">
       <string>图形数据</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,15">
       <item>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>开始日期</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="CDateEdit" name="stopdate_edt" native="true">
           <property name="styleSheet">
            <string notr="true">QDateEdit
{
    border: 1px solid #dbdbdb;
    color: #666666;
    font: 30px &quot;微软雅黑&quot;;
}

QDateEdit:on { /* shift the text when the popup opens */
    padding-top: 3px;
    padding-left: 4px;
}

QDateEdit::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 40px;

    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}

QDateEdit::down-arrow {
    image: url(:/icon/icon/calendar.ico);
}

QDateEdit::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="CDateEdit" name="startdate_edt" native="true">
           <property name="styleSheet">
            <string notr="true">QDateEdit
{
    border: 1px solid #dbdbdb;
    color: #666666;
    font: 30px &quot;微软雅黑&quot;;
}

QDateEdit:on { /* shift the text when the popup opens */
    padding-top: 3px;
    padding-left: 4px;
}

QDateEdit::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 40px;

    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}

QDateEdit::down-arrow {
    image: url(:/icon/icon/calendar.ico);
}

QDateEdit::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>结束日期</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QPushButton" name="select1_pb">
           <property name="text">
            <string>查询</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSplitter" name="splitter_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="draw_widget" name="total_draw_widget" native="true"/>
         <widget class="QSplitter" name="splitter_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="draw_widget" name="pv1_draw_widget" native="true"/>
           <widget class="draw_widget" name="pv3_draw_widget" native="true"/>
          </widget>
          <widget class="QSplitter" name="splitter_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="draw_widget" name="pv2_draw_widget" native="true"/>
           <widget class="draw_widget" name="pv4_draw_widget" native="true"/>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="power_tab">
      <attribute name="title">
       <string>发电数据</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="select_mi_power_widget" name="power_item_wd" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QWidget</extends>
   <header location="global">cdateedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>select_mi_power_widget</class>
   <extends>QWidget</extends>
   <header location="global">select_mi_power_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>draw_widget</class>
   <extends>QWidget</extends>
   <header location="global">draw_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
